bin/archive-old-enquiries

```diff
@@ -38,6 +38,8 @@ my ($opts, $usage) = describe_options(
         ['email-cutoff=s',     "Anything before this will be closed with an email sent to the reporter" ],
         ['reports=s',          "csv file with a single column of report ids" ],
     ], required => 1 } ],
+    ['council_name=s', "Name of council for templates", { required => 1 }],
+    ['council_url=s', "uri for council for templates. Omitting this flag will use cobrand->base_url"],
```

Comment:
Is there ever a time we don't want to use cobrand->base_url/base_url_for_report?